fix: increase length for value field in sms parameter

kalungia opened the pull request:
<!--

Some key notes before you open a PR:

 1. Select which branch should this PR be merged in?
 2. PR name follows [convention](http://karma-runner.github.io/4.0/dev/git-commit-msg.html)
 3. All tests pass locally, UI and Unit tests
 4. All business logic and validations must be on the server-side
 5. Update necessary Documentation
 6. Put `closes #27857` in your comment to auto-close the issue that your PR fixes


Also, if you're new here

- Documentation Guidelines => https://github.com/frappe/erpnext/wiki/Updating-Documentation

- Contribution Guide => https://github.com/frappe/frappe/blob/develop/.github/CONTRIBUTING.md

- Pull Request Checklist => https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist

-->

> Please provide enough information so that others can review your pull request:

Have changed the the value field in the child table of sms parameter to varchar(255) from 140

> Explain the **details** for making this change. What existing problem does the pull request solve?

Some sms provider's api key are beyond 140 characters, allowing 255 data long will cater for users that are with providers that go beyond the length of 140.

closes #27857

kalungia committed:
fix: increase length for value field in sms parameter

kalungia committed:
fix:(Pre-commit) failing No newline at end of file

kalungia committed:
Merge branch 'develop' into develop

blaggacao commented:
You probably should also change the column value on the doctype.

kalungia committed:
fix: change also the column value on the doctype of sms parameter

kalungia commented:
> You probably should also change the column value on the doctype.

Done

blaggacao commented:
@kalungia Thanks, can you quickly verify if the patch now becomes a no-op or is redundant, and share the results? If it's redundant, please remove to not pretend otherwise through its presence.

kalungia committed:
fix: change_value_field_length_for_sms_parameter patch to be in pre_model_sync

kalungia committed:
fix: value column length and not parameter

surajshetty3416 commented on frappe/core/doctype/sms_parameter/sms_parameter.json:
```diff
@@ -20,6 +20,7 @@
    "width": "150px"
   },
   {
+   "columns": 255,
```
@kalungia This is the wrong field to update. Update "Length" of the field to increase the limit

kalungia commented:
Keep open

kalungia committed:
fix: Update "Length" of the field to increase the limit

@surajshetty3416
https://github.com/frappe/frappe/pull/27859/files/e5aaceee502db9a8b5e6e5ac7cb2f9780f6ca586#r1828683256

akhilnarang committed:
Merge remote-tracking branch 'upstream/develop' into kalungia/develop

* upstream/develop: (289 commits)
  fix: finer grained test categorization of legacy frappe test case (#28387)
  fix(change_log): skip checking if `develop` is part of the detected version
  ci: smooth transition when adding mypy support for old PRs (#28442)
  docs: docstring on simple singledipatch util (#28437)
  chore: format
  chore: "or" over "if not" pattern
  fix: check app logo from website settings before checking navbar settings
  chore(newsletter): make attachments public
  fix: Russian translations
  fix: Polish translations
  fix: Chinese Simplified translations
  fix: Hungarian translations
  fix: Esperanto translations
  fix: Bosnian translations
  fix: Persian translations
  fix: Turkish translations
  fix: Swedish translations
  fix: German translations
  fix: Arabic translations
  fix: Spanish translations
  ...

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

akhilnarang committed:
chore: drop redundant patch

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

kalungia commented:
Keep open

akhilnarang commented:
@mergifyio rebase
